js_modules/dagit/packages/ui/src/components/MetadataTable.tsx

```diff
@@ -5,11 +5,11 @@ import {Box} from './Box';
 import {Colors} from './Colors';
 import {Table, TableProps} from './Table';
 
-type Row = {key: string; value: React.ReactNode};
+export type MetadataTableRow = {key: string; label?: React.ReactNode; value: React.ReactNode};
```

Comment:
Revert the changes in here?

Reply:
I need this to allow rendering the Info icon. Current key can only be a string since it's used as the `key` on the react node.